shared/rakefiles/xk.rake

```diff
@@ -19,6 +19,11 @@ task :xk, [:cmd, :skip_secret_mgmt, :preserve_stderr] => [:configure_serviceacco
 
   Secrets.set_secrets(@secrets)
 
+  # This is a temporary "hack", to be removed once all our clusters are
+  # migrated to the regional ones.
```

Comment:
Can you please add ticket reference to the comment?

Reply:
Sure, will do.

Reply:
Thanks!